lib/main_common.pm

```diff
@@ -1722,14 +1722,12 @@ sub load_extra_tests_filesystem {
 }
 
 sub get_wicked_tests {
-    my (%args)     = @_;
-    my $basedir    = $bmwqemu::vars{CASEDIR} . '/tests/';
-    my $wicked_dir = $basedir . 'wicked/';
-    my $suite      = get_required_var('WICKED');
-    my $type       = get_required_var('IS_WICKED_REF') ? 'ref' : 'sut';
+    my (%args)  = @_;
+    my $basedir = $bmwqemu::vars{CASEDIR} . '/tests/';
+    my $suite   = get_required_var('WICKED');
     my $exclude   = get_var('WICKED_EXCLUDE', '$a');
-    my $suite_dir = $wicked_dir . $suite . '/';
-    my $tests_dir = $suite_dir . $type . '/';
+    my $suite_dir = $basedir . 'wicked/' . $suite . '/';
```

Comment:
```suggestion
    my $suite_dir = $basedir . 'wicked/' . get_required_var('WICKED') . '/';
```

and remove line `my $suite = ...`

Reply:
also thought about that but we use this variable also in die error message so it looks nicer to keep it

Reply:
ok, right, it was hidden.

Reply:
why not remove $suite from die error message, and only leave $suite_dir